dependencies.xml

```diff
@@ -68,7 +68,7 @@ Note all install methods after "main" take
     <dask source="pip" pip_extra="[complete]"/>
     <ray source="pip" pip_extra="[default]">2.6</ray>
     <!-- redis is needed by ray, but on windows, this seems to need to be explicitly stated -->
-    <redis source="pip" os='windows'/>
+    <redis source="pip" os='windows'>5.1</redis>
```

Comment:
This is required because redis released a new version on Oct 24 2024 (5.2) and this new version is not compatible with ray 2.6 (causing the Windows test to fail).

See https://pypi.org/project/redis/#history

Reply:
well changing the dependency does not help.... @joshua-cogliati-inl no idea